Deployments/azuredeploy.bicep

```diff
@@ -104,8 +104,12 @@ resource adt 'Microsoft.DigitalTwins/digitalTwinsInstances@2021-06-30-preview' =
   Outputs
 ******************
 */
-output digitalTwinName string = adt.name
+
 output digitalTwinHostname string = adt.properties.hostName
+output storageAccountKey string = 'DefaultEndpointsProtocol=https;AccountName=${stg.name};EndpointSuffix=${environment().suffixes.storage};AccountKey=${listKeys(stg.id, stg.apiVersion).keys[0].value}'
```

Comment:
You can do this using `stg.listKeys()` instead which removes the need for the API version

Reply:
Done!